packages/fast-components-react-msft/src/action-toggle/action-toggle.spec.tsx

```diff
@@ -0,0 +1,291 @@
+            shallow(
+                <MSFTActionToggle selectedLabel="selected" unselectedLabel="unselected" />
+            );
+        }).not.toThrow();
+    });
+
+    test("should create unselectedglyph element passed in", () => {
+        const props: ActionToggleHandledProps = {
+            selectedLabel: "selected",
+            unselectedLabel: "unselected",
+            selectedGlyph: (className?: string): React.ReactNode => {
+                return <div>X</div>;
+            },
+            unselectedGlyph: (className?: string): React.ReactNode => {
+                return <div>Y</div>;
+            },
+        };
+
+        const rendered: any = mount(<ActionToggle {...props} />);
+
+        expect(rendered.contains(<div>Y</div>)).toEqual(true);
+    });
+
+    test("should create selectedglyph element passed in", () => {
+        const props: ActionToggleHandledProps = {
+            selectedLabel: "selected",
+            unselectedLabel: "unselected",
+            selectedGlyph: (className?: string): React.ReactNode => {
+                return <div>X</div>;
+            },
+            unselectedGlyph: (className?: string): React.ReactNode => {
+                return <div>Y</div>;
+            },
+            selected: true,
+        };
+
+        const rendered: any = mount(<ActionToggle {...props} />);
+
+        expect(rendered.contains(<div>X</div>)).toEqual(true);
+    });
+
+    test("should implement unhandledProps", () => {
+        const handledProps: ActionToggleProps = {
+            managedClasses,
+            href,
+            selectedLabel: "selected",
+            unselectedLabel: "unselected",
+        };
+
+        const unhandledProps: ActionToggleUnhandledProps = {
+            "aria-label": "label",
+        };
+
+        const props: ActionToggleProps = { ...handledProps, ...unhandledProps };
+
+        const rendered: any = mount(<ActionToggle {...props} />);
+
+        expect(rendered.first().prop("aria-label")).toEqual("label");
+    });
+
+    test("should apply a 'disabled' html class when prop disabled is passed", () => {
+        const props: ActionToggleHandledProps = {
+            selectedLabel: "selected",
+            unselectedLabel: "unselected",
+            disabled: true,
+        };
+
+        const rendered: any = mount(<ActionToggle {...props} />);
+
+        expect(rendered.find("button").prop("className")).toContain(
+            "actionToggle__disabled"
+        );
+    });
+
+    test("should apply a custom class-name", () => {
+        const props: ActionToggleHandledProps = {
+            selectedLabel: "selected",
+            unselectedLabel: "unselected",
+        };
+
+        const rendered: any = mount(
+            <ActionToggle className={"custom-class-name"} {...props} />
+        );
+
+        expect(rendered.find("button").prop("className")).toContain("custom-class-name");
+    });
+
+    test("should apply the custom selected ARIA label", () => {
```

Comment:
This test description is wrong. And there is some copy/paste errors in the render.

Reply:
Not seeing resolution or reasoning for not resolving

Reply:
Updated with changes.